netbox/dcim/choices.py

```diff
@@ -686,6 +686,7 @@ class InterfaceTypeChoices(ChoiceSet):
     TYPE_8GFC_SFP_PLUS = '8gfc-sfpp'
     TYPE_16GFC_SFP_PLUS = '16gfc-sfpp'
     TYPE_32GFC_SFP28 = '32gfc-sfp28'
+    TYPE_64GFC_QSFP_PLUS = '64gfc-sfpp'
```

Comment:
The slug for this value should indicate QSFP+, not SFP+.

Reply:
Ok I will adjust that. I was following the example of the existing 128GFC QSFP28. Should that one be updated as well?

Currently it is:
TYPE_128GFC_QSFP28 = '128gfc-sfp28'

Reply:
Do **not** alter any other values, as that would break existing installations.

Reply:
I have updated the slug to be 64gfc-qsfpp now as mentioned.